docs/reference/migration/migrate_7_0/api.asciidoc

```diff
@@ -79,3 +79,11 @@ the only behavior in 8.0.0, this parameter is deprecated in 7.0.0 for removal in
 ==== The deprecated stored script contexts have now been removed
 When putting stored scripts, support for storing them with the deprecated `template` context or without a context is
 now removed. Scripts must be stored using the `script` context as mentioned in the documentation.
+
+==== Get Aliases API limitations when xpack security is enabled removed
+
+When xpack security is enabled, the Get Aliases API behaviour and response
```

Comment:
I recommend using "{security}" instead of "xpack security", since the X-Pack terminology is changing in the near future.